### ISSUE TITLE
hello, i'm read your paper recently , it's brilliant.But i met some problems when running the code.

### ISSUE THREAD

**Issue by LiaoMingZhe** [Xylon-Sean/rfnet#4]
sorry to bother you ,but I  got some problems while running the code.Have you ever meet problems like this:
RuntimeError :  Function  'SqrtBackward'  returned nan values in its 0th output.

**Comment by Xylon-Sean** [comment]
@LiaoMingZhe 
It may input a number less or equal than 0 to the `sqrt()` function.

Could you give me the position caused this error?

**Comment by LiaoMingZhe** [comment]
sys:1: RuntimeWarning: Traceback of forward call that caused the error:
  File "/home/rfnet_project/rfnet-master/train.py", line 424, in <module>
    train()
  File "/home/rfnet_project/rfnet-master/train.py", line 320, in train
    else model.criterion(endpoint)
  File "/home/rfnet_project/rfnet-master/model/rf_net_module.py", line 152, in criterion
    im2_hardloss = self.des.loss(im2_lpdes, im2_rpdes, im2_limc, im2_rimcw)
  File "/home/rfnet_project/rfnet-master/model/rf_des.py", line 68, in loss
    dist_matrix = distance_matrix_vector(anchor, positive)
  File "/home/rfnet_project/rfnet-master/utils/math_utils.py", line 18, in distance_matrix_vector
    FeatSimi_Mat = torch.sqrt(FeatSimi_Mat)  # euc [0, 2]



FeatSimi_Mat = FeatSimi_Mat.clamp(min=eps, max=4.0) make sure the number is between 1e-8 and 4








------------------ 原始邮件 ------------------
发件人: "Xuelun Shen"<notifications@github.com>; 
发送时间: 2019年8月14日(星期三) 下午2:52
收件人: "Xylon-Sean/rfnet"<rfnet@noreply.github.com>; 
抄送: "廖明哲"<346745267@qq.com>; "Mention"<mention@noreply.github.com>; 
主题: Re: [Xylon-Sean/rfnet] hello, i'm read your paper recently , it'sbrilliant.But i met some problems when running the code. (#4)




@LiaoMingZhe
 It may input a number less or equal than 0 to the sqrt() function.
 
Could you give me the position caused this error?
